schema/cassandra/temporal/versioned/v1.1/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "1.1",
```

Comment:
Why minor version? Why not `2.0`? I don't have strong opinion on this. Should we leave `2.0` for real breaking change on schema level?

Reply:
In my opinion we should use versions 1.X while we are on temporal <2.0